x-pack/plugins/session_view/public/components/detail_panel_process_tab/index.tsx

```diff
@@ -0,0 +1,261 @@
+            <span css={styles.description}>{dataOrDash(leader.entryMetaSourceIp)}</span>
+          </DetailPanelCopy>
+        ),
+      });
+    }
+    return {
+      ...leaderDescriptionListInfo[idx],
+      name: leader.name,
+      listItems,
+    };
+  });
+
+  const processArgs = processDetail.args.length
+    ? `[${processDetail.args.map((arg) => `'${arg}'`)}]`
+    : '-';
+
+  return (
+    <>
+      <DetailPanelDescriptionList
+        listItems={[
+          {
+            title: <DetailPanelListItem>id</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy textToCopy={processDetail.id}>
+                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                  {dataOrDash(processDetail.id)}
+                </EuiTextColor>
+              </DetailPanelCopy>
+            ),
+          },
+          {
+            title: <DetailPanelListItem>start</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy textToCopy={processDetail.start}>
+                <span css={styles.description}>{processDetail.start}</span>
+              </DetailPanelCopy>
+            ),
+          },
+          {
+            title: <DetailPanelListItem>end</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy textToCopy={processDetail.end}>
+                <span css={styles.description}>{processDetail.end}</span>
+              </DetailPanelCopy>
+            ),
+          },
+          {
+            title: <DetailPanelListItem>exit_code</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy textToCopy={processDetail.exit_code}>
+                <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                  {dataOrDash(processDetail.exit_code)}
+                </EuiTextColor>
+              </DetailPanelCopy>
+            ),
+          },
+          {
+            title: <DetailPanelListItem>user</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy textToCopy={processDetail.user}>
+                <span css={styles.description}>{dataOrDash(processDetail.user)}</span>
+              </DetailPanelCopy>
+            ),
+          },
+          {
+            title: <DetailPanelListItem>args</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy textToCopy={processArgs}>
+                <span css={styles.description}>{processArgs}</span>
+              </DetailPanelCopy>
+            ),
+          },
+          {
+            title: <DetailPanelListItem>executable</DetailPanelListItem>,
+            description: (
+              <DetailPanelCopy
+                textToCopy={processDetail.executable
+                  .map((execTuple) => {
+                    const [executable, eventAction] = execTuple;
+                    return `${executable} ${eventAction}`;
+                  })
+                  .join(', ')}
```

Comment:
It might make the code more readable by moving this out of the JSX and calculating this upfront. Potentially in a helper function in this file.